local_customizations/seattle_tasks.yml

```diff
@@ -75,11 +75,8 @@
 #        /opt/piecewise_web/maryland_blkgrps/cb_2015_24_bg_500k.shp
 #
 - name: Ingest census blocks to postgres
-  command: > 
-    ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln seattle_blkgrpce10
-    -nlt MultiPolygon 'PG:user=postgres dbname=piecewise'
-    /opt/piecewise_web/seattle_blkgrpce10/CENSUS10_blkgrp_WGS.shp
-    
+  command: ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln seattle_blkgrpce10 -nlt MultiPolygon 'PG:user=postgres dbname=piecewise' /opt/piecewise_web/seattle_blkgrpce10/CENSUS10_blkgrp_WGS.shp
```

Comment:
Should this point to `/opt/piecewise/local_customizations/geofiles/CENSUS10_blkgrp_WGS.shp`.

What was the reason for removing the "folded scalar" wrapping from the line?